yorkie/src/main/kotlin/dev/yorkie/document/crdt/RHTPQMapNode.kt

```diff
@@ -0,0 +1,41 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.TimeTicket
+import dev.yorkie.util.PQNode
+
+/**
+ * `RHTPQMapNode` is a node of RHTPQMap.
+ */
+internal class RHTPQMapNode(val strKey: String, value: CrdtElement) :
+    PQNode<TimeTicket, CrdtElement>(value.createdAt, value) {
+
+    /**
+     * `isRemoved` checks whether this value was removed.
+     */
+    fun isRemoved(): Boolean {
+        return value.isRemoved
+    }
+
+    /**
+     * `remove` removes a value base on removing time.
+     */
+    fun remove(removedAt: TimeTicket): Boolean {
+        return value.remove(removedAt);
+    }
+
+    companion object {
+        /**
+         * `of` creates a instance of RHTPQMapNode.
+         */
+        @JvmStatic
+        fun of(strKey: String, value: CrdtElement): RHTPQMapNode {
+            return RHTPQMapNode(strKey, value)
+        }
+    }
+
+    // TODO("It's need to check this function, whether it is enough to compare two objects.")
```

Comment:
@hackerwins 
Please review this function carefully. If the comparison isn't enough, please teach me the way.

Reply:
Since the `key` is `value.createdAt`, there is no need to compare it twice, am I right?

Reply:
I didn't check it carefully.  😢 
Fixed it : af6d0a2